README.md

```diff
@@ -18,10 +18,42 @@ $ cd docker-symmetricds
 $ docker-compose up
 ```
 
+Configuration
+-------------
+
+Configuration is available using environment variables in order to configure the symmetric container.
+
+```
+GROUP_ID: <Node Group that this Node is a member of.>
+DB_HOST: <Database host name>
+DB_TYPE: <Used to tell symmetric what JDBC driver to use. Can be mysql, postgres or oracle. Defaults to postgres.>
+DB_NAME: <Database name>
+DB_USER: <Database user>
+DB_PASS: <Database password
+USERNAME: <Username for basic auth>
+PASSWORD: <Password for basic auth>
+SYNC_URL: <URL where this Node can be contacted for synchronization. At startup and during each heartbeat, the Node updates its entry in the database with this URL>
+REGISTRATION_URL: <URL where this Node can connect for registration to receive its configuration. The registration server is part of SymmetricDS and is enabled as part of the deployment>
+HTTPS: <A flag to allow TLS termination. Defaults to TRUE.>
+HTTPS_CRT: <HTTPS certificate to use if terminating TLS.
+HTTPS_KEY: <Key for provided certificate.>
+HTTPS_CA_BUNDLE: <Certificate authority for HTTPS.>
+REPLICATE_TO: <Name of symmetric GROUP_ID to replicate to.>
+REPLICATE_TABLES: <Name of tables (space seperated) and optional columns to replicate (columns are specified with a pipe and the comma separation).
```

Comment:
I think this one will need an example. (Might be nice for the others to have examples too but it's not so important.)